docs/install.md

```diff
@@ -30,6 +30,8 @@
    git clone https://github.com/sourcegraph/deploy-sourcegraph && cd deploy-sourcegraph && git checkout ${VERSION}
    ```
 
+1. Configure the `sourcegraph` storage class for the cluster by reading through "Configure a storage class" in [docs/configure.md](../configure.md#configure-a-storage-class).
```

Comment:
`["Configure a storage class"](../configure.md#configure-a-storage-class).`

The link text should correspond to what's linked to.